packages/components/ovh-at-internet/src/constants.ts

```diff
@@ -13,6 +13,40 @@ export const AT_INTERNET_LEVEL2: Record<string, string> = {
   98: 'Manager-HostedPrivateCloud',
 };
 
+const UNIVERSES = [
+  'Dedicated',
+  'Focus',
+  'Manager',
+  'Web',
+  'Server',
+  'Hub',
+  'Creation',
+  'BareMetalCloud',
+  'HostedPrivatedCloud',
+  'PublicCloud',
+  'WebCloud',
+  'Telecom',
+  'Sunrise',
+  'Account-creation',
+];
+
+const SUBUNIVERSES = [
```

Comment:
what are you calling SUB UNIVERSES ?

it looks like universes to me

Reply:
For the new tracking's nomenclature, we need to fill the universe of the app and the sub universe.
Sub universe is the range of the product for the tracking team.

But need to be updated